client/.eslintrc.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    commonjs: true
+  },
+  globals: {
+    // Webpack plugin adds `process` global
+    process: false
```

Comment:
This probably doesn't do what expected.

http://eslint.org/docs/user-guide/configuring#specifying-globals

According to the documentation, specifying `process: false` in the globals actually defines a global called `process`! the false value just disallows overwriting it.